test/test_client.py

```diff
@@ -0,0 +1,65 @@
+# unit tests for the dts.client package
+
+import dts
+import os
+import unittest
+
+class TestClient(unittest.TestCase):
+    """Unit tests for dts.client.Client"""
+
+    def setUp(self):
+        self.token = os.getenv('DTS_KBASE_DEV_TOKEN')
+        if not self.token:
+            raise ValueError('Environment variable DTS_KBASE_DEV_TOKEN must be set!')
+        self.server = "https://lb-dts.staging.kbase.us"
+
+    def test_ctor(self):
+        client = dts.Client(api_key = self.token, server = self.server)
+        self.assertTrue(client.uri)
+        self.assertTrue(client.name)
+        self.assertTrue(client.version)
```

Comment:
these aren't really worthwhile tests -- maybe they can be improved when you move to pytest, but I would expect the tests to check the values of these attributes, not just whether they are truthy or falsy.

Reply:
Yeah, I can definitely address these when we move to pytest.

Reply:
pytest uses `assert` so these would be converted to something like:
```py

assert client.uri == "some_client_uri"
assert client.name
assert isinstance(client.version, str)
assert len(client.name) > 15
```